bdk-ffi/src/bdk.udl

```diff
@@ -456,6 +463,8 @@ interface TxBuilder {
 
   TxBuilder add_utxo(OutPoint outpoint);
 
+  TxBuilder policy_path(record<string, sequence<u64>> policy_path, KeychainKind keychain);
```

Comment:
I think this is our first use of `record` https://mozilla.github.io/uniffi-rs/0.27/udl/builtin_types.html in the UDL 

Looks good https://docs.rs/bdk_wallet/latest/bdk_wallet/tx_builder/struct.TxBuilder.html#method.policy_path

Reply:
Yes, thanks!